discord/ext/converters/mixins.py

```diff
@@ -0,0 +1,15 @@
+from discord.ext.commands.bot import BotBase
+
+from .custom_converters import CONVERTERS
+
+
+class ConvertersMixin(BotBase):
+
+    def __init__(self, *args, **kwargs):
+        self.converters = {type: converter for type, converter in CONVERTERS}
+
+        super().__init__(*args, **kwargs)
+
+    async def _actual_conversion(self, ctx, converter, argument, param):
```

Comment:
This isn't a method of bot it's on Command 🤔

Reply:
This is true. Lemme fix that.